### PR TITLE
Feat/near parkings

### PR DESCRIPTION
**Pull request by JuanLugoS**
added caching, storage an BQ
added notification card that let the user know if there is an upcoming or current reservation 
imagen

### CODE REVIEW

**Comment by JuanReyes01** [comment]
Present the information in the same language of the app for consistency

File: lib/ui/home/home_page.dart [hunk 1]
```diff
@@ -279,7 +382,9 @@ class fastActionMenu extends StatelessWidget {
                           colorB1: colorB1,
                           buscar: Icons.search,
                           colorB3: colorB3,
-                          colorY1: colorY1),
+                          colorY1: colorY1,
+                          favorite: 'search',
+                          storage: storage),
                       textFastActions(
                           texto: "Buscar", colorB1: colorB1, tamanioFuente: 12),
```
Comment (JuanReyes01) [inline lib/ui/home/home_page.dart]: Spanish

File: lib/ui/home/home_page.dart [hunk 2]
```diff
@@ -290,7 +395,9 @@ class fastActionMenu extends StatelessWidget {
                           colorB1: colorB1,
                           buscar: Icons.work,
                           colorB3: colorB3,
-                          colorY1: colorY1),
+                          colorY1: colorY1,
+                          favorite: 'work',
+                          storage: storage),
                       textFastActions(
                           texto: "Trabajo",
```
Comment (JuanReyes01) [inline lib/ui/home/home_page.dart]: Spanish

File: lib/ui/home/home_page.dart [hunk 3]
```diff
@@ -303,7 +410,9 @@ class fastActionMenu extends StatelessWidget {
                           colorB1: colorB1,
                           buscar: Icons.star,
                           colorB3: colorB3,
-                          colorY1: colorY1),
+                          colorY1: colorY1,
+                          favorite: 'favorite',
+                          storage: storage),
                       textFastActions(
                           texto: "Favoritos",
```
Comment (JuanReyes01) [inline lib/ui/home/home_page.dart]: Spanish

File: lib/ui/home/home_page.dart [hunk 4]
```diff
@@ -316,7 +425,9 @@ class fastActionMenu extends StatelessWidget {
                           colorB1: colorB1,
                           buscar: Icons.people,
                           colorB3: colorB3,
-                          colorY1: colorY1),
+                          colorY1: colorY1,
+                          favorite: 'recomended',
+                          storage: storage,),
                       textFastActions(
                           texto: "Recomendados",
```
Comment (JuanReyes01) [inline lib/ui/home/home_page.dart]: Spanish

File: lib/ui/home/near_parkings.dart [hunk 5]
```diff
@@ -4,70 +4,103 @@ import 'package:flutter/material.dart';
...
       });
     });
-
   }
 
   @override
   Widget build(BuildContext context) {
-    _onListCreated();
+    late Marker choosed = Marker(
+      markerId: MarkerId("Ups! No tienes conexion a internet"),
```
Comment (JuanReyes01) [inline lib/ui/home/near_parkings.dart]: Spanish

File: lib/ui/home/near_parkings.dart [hunk 6]
```diff
@@ -187,17 +185,47 @@ class ListOfParkingLots extends StatelessWidget {
...
+  final Map choice;
+
   @override
   Widget build(BuildContext context) {
+    List<TabInfo> info = [
+      const TabInfo(text: 'ParkEz Choice', colorTab: Colors.green)
+    ];
+    TileParkings choosed = TileParkings(
+        uid: "0",
+        name: "Ups! No tienes conexion a internet",
```
Comment (JuanReyes01) [inline lib/ui/home/near_parkings.dart]: Spanish

File: lib/ui/home/home_page.dart [hunk 7]
```diff
@@ -145,8 +160,16 @@ void _onHomeCreated() {
...
           ),
 
-        fastActionMenu(colorB1: colorB1, colorB3: colorB3, colorY1: colorY1),
-        ubicationCard(fullAdress: fullAdress, colorB1: colorB1, colorB2: colorB2),
+        fastActionMenu(colorB1: colorB1, colorB3: colorB3, colorY1: colorY1, storage: storage),
+        Column(
+          children: [
+            Padding(
+              padding: const EdgeInsets.fromLTRB(0, 590, 0, 0),
+              child: reservationCard(fullAdress: "Parece que tienes una \n reserva esperandote", colorB1: colorB1, colorB2: colorB2, latitude:latitude, longitude: longitude),
```
Comment (JuanReyes01) [inline lib/ui/home/home_page.dart]: Spanish

File: lib/ui/home/home_page.dart [hunk 8]
```diff
@@ -31,26 +29,26 @@ class HomePage extends StatefulWidget {
...
   @override
   void initState() {
     super.initState();
     _onHomeCreated();
   }
 
   late GoogleMapController mapController;
-  String fullAdress = "Cargando...";
-
+  String fullAdress = "Ups!, parece que no tienes internet";
```
Comment (JuanReyes01) [inline lib/ui/home/home_page.dart]: Spanish

**Review by JuanReyes01 — CHANGES_REQUESTED** [review]
Present the information in the same language for consistency

File: lib/ui/home/home_page.dart (hunk 1 above)
Comment (JuanLugoS) [reply lib/ui/home/home_page.dart]: fixed

File: lib/ui/home/home_page.dart (hunk 2 above)
Comment (JuanLugoS) [reply lib/ui/home/home_page.dart]: fixed

File: lib/ui/home/home_page.dart (hunk 3 above)
Comment (JuanLugoS) [reply lib/ui/home/home_page.dart]: fixed

File: lib/ui/home/home_page.dart (hunk 4 above)
Comment (JuanLugoS) [reply lib/ui/home/home_page.dart]: fixed

File: lib/ui/home/near_parkings.dart (hunk 5 above)
Comment (JuanLugoS) [reply lib/ui/home/near_parkings.dart]: fixed

File: lib/ui/home/near_parkings.dart (hunk 6 above)
Comment (JuanLugoS) [reply lib/ui/home/near_parkings.dart]: fixed

File: lib/ui/home/home_page.dart (hunk 8 above)
Comment (JuanLugoS) [reply lib/ui/home/home_page.dart]: fixed

File: lib/ui/home/home_page.dart (hunk 7 above)
Comment (JuanLugoS) [reply lib/ui/home/home_page.dart]: fixed

**Review by JuanLugoS — COMMENTED** [review]
fixed

**Pull request merged**